assets/base/config.json

```diff
@@ -1,5 +1,6 @@
 {
     "DefaultServerUrl": "",
+    "DefaultServerName": "",
```

Comment:
the local configuration to set a default server name. If you haven't you could use this to speed up connecting to a server, there are other interesting stuff in here, most of them for the future or maybe they'll get removed.

To avoid having to revert changes to this file, just set the values in `/dist/assets/config.json` instead which is included in .gitignore